models/user/src/mc-user/id-token-user-info/generator.ts

```diff
@@ -0,0 +1,20 @@
+import { fake, Generator } from '@commercetools-test-data/core';
+import { createRelatedDates } from '@commercetools-test-data/utils';
+import type { TIdTokenUserInfo } from '../types';
+
+const [_, getNewerDate] = createRelatedDates();
+
+const generator = Generator<TIdTokenUserInfo>({
+  fields: {
+    iss: fake((f) => f.internet.url()),
+    sub: fake((f) => f.internet.email()),
+    aud: fake((f) => f.internet.url()),
+    exp: fake(getNewerDate),
+    iat: fake(getNewerDate),
+    email: fake((f) => f.internet.email()),
+    name: fake((f) => f.person.firstName()),
+    additionalClaims: fake((f) => f.lorem.words()),
```

Comment:
Let's leave it `null` as default

```suggestion
    additionalClaims: null,
```

Reply:
updated [32e29bc](https://github.com/commercetools/test-data/pull/466/commits/32e29bc0354cb2c9fe6144c2f029a3dd45f5b6cb)